scripts/utils.py

```diff
@@ -60,11 +53,18 @@ def get_test_targets(test_suite_path: str):
   # We use two queries here because the return code for a failed query is
   # unfortunately the same as the return code for a bazel configuration error.
   target_dir = test_suite_path.split(':')[0]
-  query = ['bazel', 'query', f'{target_dir}/...']
-  targets = check_and_get_output_lines(query, stderr_filters=BAZEL_FILTERS)
+  query = [
+      'bazel', 'query', '--ui_event_filters=-DEBUG',
+      '--noshow_loading_progress', '--noshow_progress', f'{target_dir}/...'
+  ]
+  targets = check_and_get_output_lines(query)
   if test_suite_path not in targets:
     return []
 
-  query = ['bazel', 'query', f'tests({test_suite_path})']
-  tests = check_and_get_output_lines(query, stderr_filters=BAZEL_FILTERS)
+  query = [
+      'bazel', 'query', '--ui_event_filters=-DEBUG',
+      '--noshow_loading_progress', '--noshow_progress',
+      f'tests({test_suite_path})'
+  ]
```

Comment:
Nit: This is pretty clunky, and if we're going to repeat it it might as well have a variable.

```suggestion
  query = [
      'bazel', 'query', *BAZEL_SHHHHH, f'tests({test_suite_path})'
  ]
```

Reply:
Hmmm I'm not sure the extra indirection and python argument expansion actually makes things clearer for two uses in the same function :grin:

Reply:
Either way! I was leaning towards a variable since making `bazel` be quiet isn't semantically related to the commands that are being ran, but the code is very short.